libpromises/conversion.c

```diff
@@ -1190,10 +1190,10 @@ gid_t Str2Gid(const char *gidbuff, char *groupcopy, const Promise *pp)
         }
         else if ((gr = getgrnam(gidbuff)) == NULL)
         {
-            Log(LOG_LEVEL_INFO, "Unknown group '%s' in promise", gidbuff);
 
             if (pp)
             {
+                Log(LOG_LEVEL_INFO, "Unknown group '%s' in promise", gidbuff);
```

Comment:
Maybe drop the *in promise* part?

Reply:
FWIW, the promise is logged in the next line with PromiseRef() so it seems appropriate to link them together somehow with the "in promise", maybe adding a ":" at the end to indicate "which promise is on the next line"?

Reply:
There is actually a worse situation I'd say, here: 
```
1119                    Log(LOG_LEVEL_INFO, "Unknown user in promise '%s'", ip->name);
1120
1121                    if (pp != NULL)
1122                    {
1123                        PromiseRef(LOG_LEVEL_INFO, pp);
1124                    }
```

Because the string `ip->name` seems like it would be the name of the "promise"? When that is really the name of a network user and not necessarily the requested promise.